merino-settings/Cargo.toml

```
@@ -5,9 +5,11 @@ edition = "2018"
 
 [dependencies]
 anyhow = "1.0"
-config = "0.11.0"
+async-trait = "0.1"
+config = { git = "https://github.com/mehcode/config-rs", branch = "master" }
```

Looks great but I think we should pin this in case their master breaks on us later

```suggestion
config = { git = "https://github.com/mehcode/config-rs", rev = "53e43fb" }
```

---

Good point!